android/app/src/main/java/ru/spbstu/amd/learnbraille/screens/practice/PracticeFragment.kt

```diff
@@ -71,7 +91,10 @@ class PracticeFragment : Fragment() {
                 return@Observer
             }
 
-            Toast.makeText(context, "Correct!", Toast.LENGTH_SHORT).show()
+            var toastMessage = "Правильно!"
+            if (viewModel.ifHintUsed())

```

Comment:
We always use braces accordion to the code style

Reply:
got it
maybe later we should explicitly mention it in our code style doc